openmicroscopy/openmicroscopy
# Channel processor

# What this PR does

This PR sets the default color in RGB in sequence when more than 3 channels
instead of RGBBBB etc.

# Testing this PR

 * Integration test checking that the values are correctly set when using the rendering service
 * unit test for channel processor change 
 * Import one of the images below and check that the channels are RGB RGB
 * Open and image and change the color. Save the settings. Reset the default


# Related reading

See https://www.openmicroscopy.org/community/viewtopic.php?f=13&t=8210
and https://www.openmicroscopy.org/community/viewtopic.php?f=13&t=8211

cc @sbesson @bramalingam
